backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -190,10 +191,17 @@ public void removePatientFromQueue(String patientId) {
     _repo.save(order);
   }
 
+  private void assertPatientInCurrentOrg(String patientId) {
```

Comment:
What you actually want to do here is `if (!patient.getOrganization().getInternalId().equals(getCurrentOrganization().getInternalId()) { throw... }`